CRM/Civicase/Hook/PreProcess/AddCaseAdminSettings.php

```diff
@@ -44,6 +68,52 @@ private function addCivicaseSettingsToForm(array &$settings) {
     foreach ($settingKeys as $settingKey) {
       $settings[$settingKey] = CRM_Core_BAO_Setting::SYSTEM_PREFERENCES_NAME;
     }
+    $settings = $this->changeOrderForSingleCaseRoleSetting($settings);
+  }
+
+  /**
+   * Change order for single case role setting.
+   *
+   * @param array $settings
+   *   Settings array.
+   *
+   * @return array
+   *   Settings array with changed order.
+   */
+  private function changeOrderForSingleCaseRoleSetting(array $settings) {
+    if (!empty($settings[self::CIVICASE_SINGLE_CASE_ROLE_PER_TYPE]) &&
+      !empty($settings[self::CIVICASE_ALLOW_MULTIPLE_CLIENTS])) {
+      $newSettings = [];
+      foreach ($settings as $k => $val) {
+        if ($k === self::CIVICASE_SINGLE_CASE_ROLE_PER_TYPE) {
+          continue;
+        }
+        $newSettings[$k] = $val;
+        if ($k === self::CIVICASE_ALLOW_MULTIPLE_CLIENTS) {
+          $newSettings[self::CIVICASE_SINGLE_CASE_ROLE_PER_TYPE] = $settings[self::CIVICASE_SINGLE_CASE_ROLE_PER_TYPE];
+        }
+      }
+      return $newSettings;
+    }
+
+    return $settings;
+  }
+
+  /**
+   * Adds the default multiple case client to the form attributes.
+   *
+   * @param CRM_Core_Form $form
+   *   Form object class.
+   */
+  private function addDefaultMultipleCaseClientToForm(CRM_Core_Form $form) {
+    $attributes = $form->getAttributes();
+    $xml = $this->xmlProcessor->retrieve("Settings");
+    $allowMultipleClients = 0;
+    if (!empty($xml->AllowMultipleCaseClients)) {
+      $allowMultipleClients = $xml->AllowMultipleCaseClients->__toString();
+    }
+    $attributes['defaultMultipleCaseClient'] = $allowMultipleClients;
+    $form->setAttributes($attributes);
```

Comment:
Rather than setting this attribute on the form itself seems it seems unrelated. I think we can set it in the `civicaseAllowMultipleClients`  element itself. See this https://github.com/compucorp/uk.co.compucorp.civicase/blob/509a3d8e2b990285234d651bc6eb10868d024f8c/settings/CiviCase.setting.php#L61-L64

We can modify this function to return the default multiclient case value and then set the attribute in the `changeOrderForSingleCaseRoleSetting` function.